journals/apps/journals/models.py

```diff
@@ -698,19 +701,25 @@ def get_user_pages(user, pages=None):
         return user_pages
 
     @staticmethod
-    def get_user_collections(user, collections=None):
+    def get_user_collections(user, collection_type, collections=None):
         """
         Args:
             user: instance of User model
+            collection_type: string for filtering collections base on their type
             collections: queryset of collections to filter
         Returns: wagtail collections queryset where given user has add, change permissions
         if collections queryset is provided filter is applied on that.
         """
+        model_mapping = {
+            'documents': {"model": Document, "owner_field": "uploaded_by_user"},
+            'images': {"model": Image, "owner_field": "uploaded_by_user"},
```

Comment:
I'm creating a `JournalImage` subclass in another PR. Will I need to change this afer that lands?